.travis.yml

```diff
@@ -5,7 +5,7 @@ node_js:
 - '6'
 - '8'
 script:
-- ./bin/run-test.sh -c
+- gulp && ./bin/run-test.sh -c
```

Comment:
what version of gulp would that run?  Shouldn't this be npm run build or npx gulp?